deployment/ansible/roles/nginx-certbot-deploy/tasks/configure_domain.yml

```diff
@@ -41,6 +41,21 @@
   when: not letsencrypt_certificate.stat.exists
   notify: Reload nginx
 
+- name: Set up basic auth
+  when: domain.auth_basic_user is defined and domain.auth_basic_user != ""
+  block:
+    - name: Install passlib
+      package:
+        name: python-passlib
+        state: present
+
+    - name: Generate basic auth
+      htpasswd:
+        path: /etc/nginx/passwords/{{ domain.name | mandatory }}
+        name: "{{ domain.auth_basic_user }}"
+        password: "{{ domain.auth_basic_pass }}"
+      notify: Restart Nginx
```

Comment:
```
ERROR! The requested handler 'Restart Nginx' was not found in either the main handlers list nor in the listening handlers list
```

Mungkin pakai `Reload nginx` aja kali ya biar ngikut yang udah ada di file ini?

Reply:
Let me check, butuh restart atau cukup reload aja

Reply:
Barusan gw coba sih Reload bisa somehow.

Reply:
Resolved in 911efe4096de7f38c5ab11b554f76ab81aca4f06